### PR TITLE
Upgrade to bootstrap 5 and remove material design

### PR DESCRIPTION
**Pull request by sebastianrakel**
Did not found a bootstrap 5 material design which is not mdbootstrap

screen_blog
screen_coc
screen_docs
screen_elec
screen_plug
screen_prvi
screen_sec
screen_sponsors

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -12,9 +12,7 @@
     <link rel="shortcut icon" href="{{site.url}}{{site.baseurl}}/favicon.ico" type="image/x-icon">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/0.5.10/css/bootstrap-material-design.min.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/0.5.10/css/ripples.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```
Comment (bastelfreak) [inline _includes/head.html:15]: we need something to automatically update this in the future :D

**Review by bastelfreak — APPROVED** [review]

File: _includes/head.html [hunk 2]
```diff
@@ -12,9 +12,7 @@
     <link rel="shortcut icon" href="{{site.url}}{{site.baseurl}}/favicon.ico" type="image/x-icon">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (ekohl) [inline _includes/head.html:14]: Is this still needed if we're dropping material design?

File: plugins/index.md [hunk 3]
```diff
@@ -11,14 +11,14 @@ Tools without plugins are going to be listed in the table under the `Tools`
...
 
-<div class="panel panel-info">
-  <div class="panel-heading">Table of Contents</div>
+<div class="card">
+  <div class="card-header panel-heading bg-primary text-white">Table of Contents</div>
   <div class="panel-body">
-  <ul>
-  <li><a href="#tools">Tools</a></li>
+	<ul class="list-group">
+	<li class="list-group-item"><a href="#tools">Tools</a></li>
```
Comment (ekohl) [inline plugins/index.md:18]: Nit: the rest uses spaces and this uses tabs

**Review by ekohl — COMMENTED** [review]
Looks good. Some minor comments inline.

**Review by bastelfreak — APPROVED** [review]

File: _includes/head.html (hunk 2 above)
Comment (sebastianrakel) [reply _includes/head.html]: i dropped them also. looks good

File: plugins/index.md (hunk 3 above)
Comment (sebastianrakel) [reply plugins/index.md]: fixed it

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**